Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.h

```diff
@@ -27,6 +27,7 @@
 
 #if ENABLE(MEDIA_SOURCE) && USE(AVFOUNDATION)
 
+#include "AudioVideoRendererAVFObjC.h"
```

Comment:
This doesn't appear to be needed.

Reply:
it's needed for the TrackIdentifier used in the maps.